src/olympia/reviewers/tests/test_utils.py

```diff
@@ -371,15 +371,16 @@ def test_actions_public_static_theme(self):
         # action, but it should not be available for static themes.
         self.grant_permission(self.request.user, 'Addons:PostReview')
         self.addon.update(type=amo.ADDON_STATICTHEME)
-        expected = []
+        expected = ['reject_multiple_versions']
```

Comment:
I made PR by referring to the guide.
According to the test code, It cannot be used `reject_multiple_versions` if `PostReview` and `static themes`.
Should I make an exception to limit using `reject_multiple_versions` in this case?

Reply:
You should need `Addons:ThemeReview` to deal with static themes in general. Someone with just `Addons:PostReview` shouldn't be able to reject themes.